Factory and Teamcamp integrations (#13365)
getsentry/sentry-docs: docs/organization/integrations/issue-tracking/factory/index.mdx

---
title: Factory
sidebar_order: 1
description: "Learn about Sentry's Factory integration and how it uses AI to help fix issues."
---

> Factory integrates with a users codebase, and pulls in Issues to help automate detecting and fixing issues.

Reviewer: ```suggestion
Factory integrates with your codebase, and pulls in Issues to help automate detecting and fixing issues.
```
